test/fixtures/books.yml

```diff
@@ -0,0 +1,17 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
The best practice is to use seeds and factories instead of fixtures. We will see this more in detail in the future!